vm/nova/vm.go

```diff
@@ -301,97 +307,15 @@ func accountStateChangeValid(vmParams *vm.Params, input *vm.ChainOutputWithIDs,
 		}
 	}
 
-	if current.StateIndex == next.StateIndex {
-		return accountGovernanceSTVF(vmParams, input, next)
-	}
-
-	return accountStateSTVF(vmParams, input, next)
-}
-
-func accountGovernanceSTVF(vmParams *vm.Params, input *vm.ChainOutputWithIDs, next *iotago.AccountOutput) error {
-	//nolint:forcetypeassert // we can safely assume that this is an AccountOutput
-	current := input.Output.(*iotago.AccountOutput)
-
-	switch {
-	case current.Amount != next.Amount:
```

Comment:
@cyberphysic4l @PhilippGackstatter please have a closer look at those change, because I had to merge the governor and state transition logic here.